phpunit.xml.dist

```diff
@@ -11,6 +11,7 @@
     <php>
         <server name="KERNEL_CLASS" value="Liip\Acme\Tests\App\AppKernel" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak"/>
+        <env name="MONGODB_URL" value="mongodb://mongodb:27017/test"/>
```

Comment:
The value can be removed from here and hard-coded in `tests/AppConfigMongodb/config.yml`.

Reply:
I prefer env-variables, but I've changed it.